setup.py

```diff
@@ -78,7 +78,9 @@ def package_files(directory, strip_leading):
               'pandas',
               'pyyaml',
               'plotly',
-              'imgaug'
+              'imgaug',
+              'gym',
+              'gym-donkeycar @ git+ssh://git@github.com/tawnkramer/gym-donkeycar.git@master#egg=gym-donkeycar',
```

Comment:
This doesn't work in the CI, need to fix

Reply:
Can you try removing the trailing `@master...` and just put:
'gym-donkeycar @ git+ssh://git@github.com/tawnkramer/gym-donkeycar.git`